File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/polizza.java
package progetto_terminal_marittimo;


public class polizza {
    private int ID;
    private int quantita_merce;
    private int id_viaggio;
    private int id_fornitore;
    private int id_merce;


    public polizza(int id, int q, int v, int m, int f){
        this.ID=id;
        this.quantita_merce=q;
        this.id_merce=m;
        this.id_viaggio=v;
        this.id_fornitore=f;
    }

    public int getID(){
        return this.ID;
    }

    public int getQuantita(){
        return this.quantita_merce;
    }

    public int getMerce(){
        return this.id_merce;
    }

    public int getViaggio(){
        return this.id_viaggio;
    }

    public int getFornitore(){
        return this.id_fornitore;
    }
}

File: pagine/js/polizza.js
class Polizza {
  constructor(id, quantita, viaggio, merce, fornitore) {
    this.ID = id;
    this.quantita_merce = quantita;
    this.id_viaggio = viaggio;
    this.id_merce = merce;
    this.id_fornitore = fornitore;
  }

  getID() {
    return this.ID;
  }

  getQuantita() {
    return this.quantita_merce;
  }

  getMerce() {
    return this.id_merce;
  }

  getViaggio() {
    return this.id_viaggio;
  }

  getFornitore() {
    return this.id_fornitore;
  }
}
